testsuite/MDAnalysisTests/coordinates/test_pdb.py

```diff
@@ -455,13 +455,30 @@ def get_MODEL_lines(filename):
             # test number (only last 4 digits)
             assert int(line[10:14]) == model % 10000
 
-    def test_segid_chainid(self, universe2, outfile):
-        """check whether chainID comes from last character of segid (issue #2224)"""
-        ref_id = 'E'
-        u = universe2
+    def test_chainid_validated(self, universe3, outfile):
```

Comment:
Could you please use `pytest.mark.parametrize` (https://docs.pytest.org/en/stable/parametrize.html) to turn this into three tests with different arguments? There are a few examples around if you ctrl+F in the tests.

Reply:
Good idea!